gcloud/bigtable/_helpers.py

```diff
@@ -85,8 +90,11 @@ def make_stub(client, stub_factory, host, port):
     :rtype: :class:`grpc.beta._stub._AutoIntermediary`
     :returns: The stub object used to make gRPC requests to a given API.
     """
+    root_certificates = get_certs()
+    client_credentials = implementations.ssl_client_credentials(
+        root_certificates, private_key=None, certificate_chain=None)
+    channel = implementations.secure_channel(
+        host, port, client_credentials)
     custom_metadata_transformer = MetadataTransformer(client)
-    return stub_factory(host, port,
-                        metadata_transformer=custom_metadata_transformer,
-                        secure=True,
-                        root_certificates=get_certs())
+    return stub_factory(channel, host=host,
```

Comment:
OK I just removed `host` here and re-ran the system tests in `gcloud-python-bigtable` with no issues. It must be intended as an optional over-ride for the `channel`.